app/routes/signin-oidc.js

```diff
@@ -144,7 +152,8 @@ module.exports = [{
           return h.redirect(oldClaimJourney)
         }
       } catch (err) {
-        console.error(`Received error with name ${err.name} and message ${err.message}.`)
+        request.logger.setBindings({ err })
+
```

Comment:
Good spot. Actually if that throws it takes the app down as there's nothing to catch it. Added an await and wrapped it to catch. No point in throwing further as I think the user is better to just be told they aren't eligible rather than potentially getting an error page.